.expeditor/config.yml

```diff
@@ -92,14 +107,22 @@ subscriptions:
       - built_in:trigger_omnibus_expcache_build:
           only_if_modified:
             - config/software/*
-# These actions are taken, in the order specified, when an Omnibus artifact is promoted
-# within Chef's internal artifact storage system.
-#
-# TODO: add promoted_to_unstable action to update changelog with modified omnibus components
-artifact_actions:
-  promoted_to_unstable:
-    - built_in:update_acc
-  promoted_to_current:
-  promoted_to_stable:
-    - built_in:rollover_changelog
-    - built_in:notify_chefio_slack_channels
+  # TODO: add action to update changelog with modified omnibus components
+  - workload: artifact_published:unstable:chef-server:*
+    actions:
+      - built_in:update_acc
+  - workload: artifact_published:current:chef-server:*
```

Comment:
I'm not sure what specifying this without any actions will do, but probably it will break things.